sg/makefile.go

```diff
@@ -15,7 +15,7 @@ import (
 	"go.einride.tech/sage/internal/strcase"
 )
 
-const defaultGoVersion = "1.23.4"
+const defaultGoVersion = "1.25.3"
```

Comment:
1.25.4 was released yesterday 😆

Reply:
@Tethik let's get this PR merged. But can you please set it to


```suggestion
const defaultGoVersion = "1.25.5"
```